### PR TITLE
REPO-2170: Enhance People API to return capabilities

### PR DESCRIPTION
**Pull request by CodrinChirica**
- initial implementation and test for adding capabilities to People Api

### CODE REVIEW

File: src/main/java/org/alfresco/rest/api/model/Person.java [hunk 1]
```diff
@@ -445,6 +446,14 @@ public void setAspectNames(List<String> aspectNames)
 	{
 		this.aspectNames = aspectNames;
 	}
+	
+	public Map<String, Boolean> getCapabilities() {
```
Comment (derekhulley) [inline src/main/java/org/alfresco/rest/api/model/Person.java]: Formatting

File: src/test/java/org/alfresco/rest/api/tests/TestPeople.java [hunk 2]
```diff
@@ -2296,6 +2295,42 @@ public void removeAvatar() throws IOException, PublicApiException{
         
     }
 
+    @Test
+    public void testListPeopleWithCapabilities() throws Exception
```
Comment (cristianturlica) [inline src/test/java/org/alfresco/rest/api/tests/TestPeople.java]: Please add a test or update the existing test for getPeopleById to see that we have the capabilities also there.

Comment (CodrinChirica) [reply src/test/java/org/alfresco/rest/api/tests/TestPeople.java]: Thanks for the observation. Added a validation in the existing test in this [commit ](https://github.com/Alfresco/alfresco-remote-api/pull/47/commits/bc3910ce998ca728e4a82d53a1eebe50be8b58b2).

File: src/test/java/org/alfresco/rest/api/tests/TestPeople.java [hunk 3]
```diff
@@ -236,6 +237,9 @@ public void testGetPerson() throws Exception
             publicApiClient.setRequestContext(new RequestContext(account1.getId(), person1));
             Person resp = people.getPerson(org.alfresco.rest.api.People.DEFAULT_USER);
             Person person1Entity = repoService.getPerson(person1);
+            assertNotNull("Check if capabilities are present", resp.getCapabilities());
+            assertTrue("Check if -me- user is mutable",
+                    resp.getCapabilities().get("isMutable").booleanValue());
```
Comment (cristianturlica) [inline src/test/java/org/alfresco/rest/api/tests/TestPeople.java]: Unboxing for .booleanValue() isn't needed here.

**Pull request merged**